gflownet/gflownet.py

```diff
@@ -890,10 +864,10 @@ def test(self):
             z_pred = sum([hist[tuple(x)] for x in x_tt]) + 1e-9
             density_pred = np.array([hist[tuple(x)] / z_pred for x in x_tt])
             # corr = np.corrcoef(density_pred, density_true)[0, 1]
-            corr = 0.0
+            # corr = 0.0
```

Comment:
Remove?

Reply:
Ah no, sometimes I do not want to calculate the correlation (multi-fidelity grid) as it takes too long. I will implement a sophisticated way of toggling later on, but for now I just comment and uncomment whenever necessary